sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -151,25 +153,38 @@ class SimpleSearcherLimo {
       this.expressRelayConfig.feeReceiverRelayer
     );
 
-    bid.transaction.recentBlockhash = opportunity.blockHash;
+    bid.transaction.recentBlockhash = recentBlockhash;
     bid.transaction.sign(this.searcher);
     return bid;
   }
 
   async opportunityHandler(opportunity: Opportunity) {
-    const bid = await this.generateBid(opportunity as OpportunitySvm);
-    try {
-      const bidId = await this.client.submitBid(bid);
-      console.log(
-        `Successful bid. Opportunity id ${opportunity.opportunityId} Bid id ${bidId}`
+    if (this.recentBlockhash[this.chainId]) {
```

Comment:
I think it's better to have an early return instead of if else clause
```
if (!this.recentBlockhash[this.chainId]) {
     console.log("...");
     return;
}
...
```

Reply:
is there any difference beyond readability?

Reply:
I think it'll remove one level of nesting for the main functionality which makes the code easier to follow.